affinity/cpp-20/d0796r1.md

```diff
@@ -94,6 +102,66 @@ There are some additional challenges which we have been investigating but are no
 * Migrating data from memory allocated in one partition to another
 * Defining memory placement algorithms or policies
 
+
+## Proposed Wording
+
+### Header synopsis
+
+```cpp
+namespace std {
+namespace experimental {
+namespace execution {
+
+/* Execution resource */
+
+struct execution_resource {
+
+  execution_resource() = delete;
```

Comment:
Shouldn't this have a private implementation defined constructor at least?

Reply:
An implementation may have private constructors which are implementation defined, but I don't think this needs to be specified in the synopsis, perhaps we can put this in a note in the wording? This is to state that an `execution resource` should not be default constructible, maybe we should describe this in a different way?